brian2/stateupdaters/base.py

```diff
@@ -127,10 +138,16 @@ def apply_stateupdater(equations, variables, method):
             if the_method is None:
                 raise ValueError(('No stateupdater that is suitable for the '
                                   'given equations has been found.'))
-            logger.info('Using state updater: %r' % the_method)
+            if group_name is not None:
+                msg_text = ("No numerical integration method specified for group "
+                            "'{group_name}', choosing method '{method}'.")
+            else:
+                msg_text = ("No numerical integration method specified, "
+                            "choosing method '{method}'.")
+            logger.info(msg_text.format(group_name=group_name,
+                                        method=the_method), 'method_choice')
```

Comment:
Oops, I meant don't keep it at info level - it should be at debug level. Either way, I'm happy to keep it if you think it's better like this. Your idea about the linear method taking a long time also raises another potential item for my idea of the performance checklist: if the choosing of the state updater takes more than a few seconds we could add that to the checklist? (Or just have an info/debug message about it.)